packages/renderer/src/components/Modals/InstallClient/component.tsx

```diff
@@ -0,0 +1,80 @@
+import { Box, OperativeSystem, Typography, DownloadButton } from 'decentraland-ui2';
+import {
+  DownloadButtonAppleIcon,
+  DownloadButtonWindowsIcon,
+} from 'decentraland-ui2/dist/components/DownloadButton/DownloadButton.styled';
+import { CDNSource, getCDNRelease } from 'decentraland-ui2/dist/modules/cdnReleases';
+import { useAdvancedUserAgentData } from '@dcl/hooks';
+import { actions } from '/@/modules/store/editor';
+import { Modal } from '..';
+import { Button } from '../../Button';
+import LogoDCLSVG from '/assets/images/logo-dcl.svg';
+import type { Props } from './types';
+
+import './styles.css';
+import { useCallback, useMemo } from 'react';
+import { useDispatch } from '#store';
+
+export function InstallClient({ open, onClose }: Props) {
+  const [_, agent] = useAdvancedUserAgentData();
+  const dispatch = useDispatch();
+
+  const handleDownload = useCallback(async () => {
+    if (!agent) return;
+    const cdn = getCDNRelease(CDNSource.LAUNCHER);
```

Comment:
We should use the same logic as @cyaiox did in all the download buttons for the new launcher.